src/Microsoft.ServiceFabric.Services.Remoting/FabricTransport/Runtime/FabricTransportRemotingListenerSettings.cs

```diff
@@ -173,6 +200,29 @@ public bool UseWrappedMessage
             set { this.useWrappedMessage = value; }
         }
 
+        /// <summary>
+        /// Gets or sets the depth of exceptions to be sent to the client incase of remoting call failing with exception.
+        /// </summary>
+        public int RemotingExceptionDepth
+        {
+            get
+            {
+                return this.remotingExceptionDepth;
+            }
+
+            set
+            {
+                if (value <= 0)
+                {
+                    this.remotingExceptionDepth = int.MaxValue;

```

Comment:
If user sets depth as negative number, do we plan on recursivesly serial/deserial irrespective of the depth?

Reply:
Thats correct. I have added documentation to clarify the same.